components/elm/src/biogeochem/AllocationMod.F90

```diff
@@ -1128,13 +1132,15 @@ subroutine Allocation2_ResolveNPLimit (bounds, num_soilc, filter_soilc  , &
               decompmicc(:)  =  elm_fates%fates(ci)%bc_out(s)%decompmicc(:) ! Should be (nlevdecomp)
 
               cn_scalar_ptr          => elm_fates%fates(ci)%bc_out(s)%cn_scalar          ! (i,j)
+              cn_scalar_runmean_ptr  => elm_fates%fates(ci)%bc_out(s)%cn_scalar          ! (i,j)
               plant_nh4demand_vr_ptr => plant_nh4demand_vr_fates
               km_nh4_ptr             => elm_fates%fates(ci)%bc_pconst%eca_km_nh4
               vmax_nh4_ptr           => elm_fates%fates(ci)%bc_pconst%eca_vmax_nh4
               plant_no3demand_vr_ptr => plant_no3demand_vr_fates
               km_no3_ptr             => elm_fates%fates(ci)%bc_pconst%eca_km_no3
               vmax_no3_ptr           => elm_fates%fates(ci)%bc_pconst%eca_vmax_no3
               cp_scalar_ptr          => elm_fates%fates(ci)%bc_out(s)%cp_scalar
+              cp_scalar_runmean_ptr  => elm_fates%fates(ci)%bc_out(s)%cp_scalar
```

Comment:
@rgknox It appears that FATES does not have `<cn/cp>_scalar_runmean`, so I have set them to `<cn/cp>_scalar`.

Reply:
It also looks like we don't need the _scalar_ptr's anymore.